core/src/components/picker-column-option/picker-column-option.tsx

```diff
@@ -60,12 +67,54 @@ export class PickerColumnOption implements ComponentInterface {
     /**
      * The initial value of `aria-label` needs to be set for
      * the first render.
+
      */
     this.ariaLabel = inheritedAttributes['aria-label'] || null;
   }
 
+  connectedCallback() {
+    this.pickerColumn = this.el.closest('ion-picker-column');
+  }
+
+  disconnectedCallback() {
+    this.pickerColumn = null;
+  }
+
+  /**
+   * The column options can load at any time
+   * so the options needs to tell the
+   * parent picker column when it is loaded
+   * so the picker column can ensure it is
+   * centered in the view.
+   *
+   * We intentionally run this for every
+   * option. If we only ran this from
+   * the selected option then if the newly
+   * loaded options were not selected then
+   * scrollActiveItemIntoView would not be called.
+   */
+  componentDidLoad() {
+    const { pickerColumn } = this;
+    if (pickerColumn !== null) {
+      pickerColumn.scrollActiveItemIntoView();
+    }
+  }
+
+  /**
+   * When an option is clicked update the
+   * parent picker column value. This
+   * component will handle centering the option
+   * in the column view.
+   */
+  onClick() {
```

Comment:
When clicking an option to select it (vs. scrolling to it), I'm no longer getting a smooth scroll animation; it just snaps right to the option. Is that expected?

Reply:
good catch! fixed in https://github.com/ionic-team/ionic-framework/pull/28622/commits/da2db5ca63f4a773682ddb311086d5b73281e571